pkg/customresource/customresource.go

```diff
@@ -65,6 +65,7 @@ type Context struct {
 	APIExtensionClientset apiextensionsclient.Interface
 	Interval              time.Duration
 	Timeout               time.Duration
+	ParentContext         contextpkg.Context
```

Comment:
```suggestion
	Context         contextpkg.Context
```
Better to call it just context

Reply:
Done